go/vt/vttablet/grpcqueryservice/server.go

```diff
@@ -248,8 +249,8 @@ func (q *query) BeginExecute(ctx context.Context, request *querypb.BeginExecuteR
 		request.EffectiveCallerId,
 		request.ImmediateCallerId,
 	)
-
-	result, transactionID, err := q.server.BeginExecute(ctx, request.Target, request.Query.Sql, request.Query.BindVariables, request.Options)
+	// ignore returned TabletAlias here, we are not sending it in the grpc response
+	result, transactionID, _, err := q.server.BeginExecute(ctx, request.Target, request.Query.Sql, request.Query.BindVariables, request.Options)
```

Comment:
Didn't we decide to send the tablet alias back from vttablet? The main difference is that we're not going to rely on it on the vtgate side, right?

Reply:
added alias to Response protos.